stdlib/3.4/enum.pyi

```diff
@@ -45,11 +45,11 @@ if sys.version_info >= (3, 6):
         def __xor__(self: _T, other: _T) -> _T: ...
         def __invert__(self: _T) -> _T: ...
 
-    # All `type: ignore` comments below due to IntFlag making the function signatures more permissive.
+    # int and Flag have some incompatible methods
```

Comment:
The error mypy produces without the ignore is
```
stdlib/3.4/enum.pyi:49: error: Definition of "__and__" in base class "int" is incompatible with definition in base class "Flag"
stdlib/3.4/enum.pyi:49: error: Definition of "__or__" in base class "int" is incompatible with definition in base class "Flag"
stdlib/3.4/enum.pyi:49: error: Definition of "__xor__" in base class "int" is incompatible with definition in base class "Flag"
stdlib/3.4/enum.pyi:49: error: Definition of "__invert__" in base class "int" is incompatible with definition in base class "Flag"
```
So I don't think it's accurate to say that the type ignore is because of more permissive signatures. I don't have a good intuition here (subclass compatibility is hard enough to understand without multiple inheritance), but I feel like mypy is perhaps too strict in this case.

Anyway, I'll clarify the comment.